calico/reference/architecture/design/l3-interconnect-fabric.md

```diff
@@ -4,264 +4,130 @@ description: Understand considerations for implementing interconnect fabrics wit
-areas, [^4] and therefore is fairly straight forward.
-
-That said, there are a few design rules for BGP that need to be kept in
-mind when designing an IP fabric that will interconnect nodes in a
-{{site.prodname}} network. These BGP design requirements *can* be worked around, if
-necessary, but doing so takes the designer out of the standard BGP
-*envelope* and should only be done by an implementer who is *very*
-comfortable with advanced BGP design.
+- A BGP fabric where each of the TOR switches (and their subsidiary compute servers) are a unique {% include open-new-window.html text='Autonomous System (AS)' url='https://en.wikipedia.org/wiki/Autonomous_System_(Internet)' %}
+and they are interconnected via either an Ethernet switching plane provided by the spine switches in a {% include open-new-window.html text='leaf/spine' url='http://bradhedlund.com/2012/10/24/video-a-basic-introduction-to-the-leafspine-data-center-networking-fabric-design/' %} architecture, or via a set of spine switches, each of which is also a unique AS. We'll refer to this as the *AS per rack* model. This model is detailed in {% include open-new-window.html text='IETF RFC 7938' url='https://datatracker.ietf.org/doc/html/rfc7938' %}.
+
+-  A BGP fabric where each of the compute servers is a unique AS, and the TOR switches make up a transit AS. We'll refer to this as the *AS per server* model.
+
+Each of these models can either have an Ethernet or IP spine. In the case of an Ethernet spine, each spine switch provides an isolated Ethernet connection *plane* as in the {{site.prodname}} Ethernet interconnect fabric model and each TOR switch is connected to each spine switch.
+
+Another model is where each spine switch is a unique AS, and each TOR switch BGP peers with each spine switch. In both cases, the TOR switches use ECMP to load-balance traffic between all available spine switches.
+
+#### BGP network design considerations
+
+Contrary to popular opinion, BGP is actually a fairly simple protocol. For example, the BGP configuration on a {{site.prodname}} compute server is approximately sixty lines long, not counting comments. The perceived complexity is due to the things that you can *do* with BGP. Many uses of BGP involve complex policy rules, where the behavior of BGP can be modified to meet technical (or business, financial, political, etc.) requirements. A default {{site.prodname}} network does not venture into those areas, ([note 4](#note-4)) and therefore is fairly straight forward.
+
+That said, there are a few design rules for BGP that need to be kept in mind when designing an IP fabric that will interconnect nodes in a {{site.prodname}} network. These BGP design requirements *can* be worked around, if necessary, but doing so takes the designer out of the standard BGP *envelope* and should only be done by an implementer who is *very* comfortable with advanced BGP design.
 
 These considerations are:
 
-AS continuity
+- AS continuity or *AS puddling* 
```

Comment:
I'm not looking at the preview here, so just wondering: Do these work better with some kind of formatting? Bold? Italic? H4/5?

Reply:
Not sure it is needed at all. I found myself correcting some things, and not others. Trying to find the right balance between total makeover and cleanup.